Events/Ch10.event

```diff
@@ -0,0 +1,231 @@
+//#include EAstdlib.event
+
+EventPointerTable(0x31,ThisChapter)
+
+EscapePointPointerTable(0x0B, Ch16EscapePoint)
+
+
+Ch16EscapePoint:
+//EscapePoint(0, 0, MoveUp)
+EscapePoint(3, 8, NoMove)
+EscapePointEnd
+
+ThisChapter:
+POIN TurnBasedEvents
+POIN CharacterBasedEvents
+POIN LocationBasedEvents
+POIN MiscBasedEvents
+POIN Dunno Dunno Dunno
+POIN Tutorial
+POIN TrapData TrapData
+POIN Units Units
+POIN $0 $0 $0 $0 $0 $0
+POIN BeginningScene EndingScene
+
+TurnBasedEvents:
+TurnEventPlayer(0,pointer,3)
```

Comment:
These do not refer to anything valid. Please change them to what they're supposed to be.